src/nvim/autocmd.c

```diff
@@ -1972,6 +1973,7 @@ static bool call_autocmd_callback(const AutoCmd *ac, const AutoPatCmd *apc)
     PUT(data, "match", CSTR_TO_OBJ(autocmd_match));
     PUT(data, "file", CSTR_TO_OBJ(autocmd_fname));
     PUT(data, "buf", INTEGER_OBJ(autocmd_bufnr));
+    PUT(data, "winid", INTEGER_OBJ(apc->arg_winid));
```

Comment:
in sympathy with `buf` (just above this line) should this be named `win` ?
```suggestion
    PUT(data, "win", INTEGER_OBJ(apc->arg_winid));
```